### DISCUSSION TITLE
Minimum tfaction example

### DISCUSSION THREAD

**Discussion by sato-s** [suzuki-shunsuke/tfaction#2059] [Deprecated (Proposal)]
### Feature Overview

tfaction example like [tfaction-example](https://github.com/suzuki-shunsuke/tfaction-example), but with minimum settings and anyone can run with just forking. 

### Why is the feature needed?

tfaction is very helpful and effective OSS. (Thanks!)
But, I found installation isn't easy and can be very fallible.  

If there is minimum tfaction example which anyone can run with just forking, it will be very helpful.
Because, people can use it as a scaffold to install tfaction in their terraform.
AWS or GCP resources isn't necessary, just creating [null_resource](https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource) would be enough.

Currently, we have [tfaction-example](https://github.com/suzuki-shunsuke/tfaction-example), But it has exhaustive configuration and is hard to re-use.

Also, this will be helpful for replicating a bug and makes bug report process easier.

**Comment by suzuki-shunsuke** [comment]
Thank you for your suggestion.

I agree that the setup of tfaction isn't easy and we should simplify it ideally.
But at the same time we want to provide sophisticated workflows too.
The minimum workflows can't be enough for production use.

1. tfaction-example uses `pull_request_target`. https://dev.to/suzukishunsuke/secure-github-actions-by-pullrequesttarget-641
2. tfaction-example split workflows using reusable workflows. This makes workflows complicated but this is useful for effective dependency updates

> AWS or GCP resources isn't necessary, just creating [null_resource](https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource) would be enough.

tfaction-example doesn't use aws and gcp providers now. It mainly uses github_issue_label and null_resource.
It uses HCP Terraform as Terraform Backend because we don't want to use S3 and GCS Backend in tfaction-example and local state isn't appropriate for CI.
Local State always causes diff, blocking auto-merge of Renovate pull requests.
That's why we use HCP Terraform as Terraform Backend, so people can't work CI by just forking tfaction-example.
Anyway, we don't assume people use example's Terraform configuration. This is just example.

> Also, this will be helpful for replicating a bug and makes bug report process easier.

Yeah, this is awesome, but actually it's not so easy.

**Reply by sato-s** [reply]
A production-ready example with renovate is of course helpful.

But I think developers usually try a new technology with their hobby project or on-the-fly experimental project.
IMO, A production-unready but simple example would attract more people.
